app/client/tests/e2e/LoggedIn.spec.ts

```diff
@@ -111,7 +111,10 @@ test.describe("Logged in Tests", () => {
         await expect(page.locator("#cy canvas")).toHaveCount(3);
         // can browse back to Home page and see new project in history
         await page.goto(config.clientUrl());
-        await expect(await page.locator(".saved-project-row").last()).toHaveText("test project", { timeout });
+        await expect(await page.locator(".saved-project-row .saved-project-name").last())
+            .toHaveText("test project", { timeout });
+        expect(await (await page.locator(".saved-project-row .saved-project-date").last()).innerText())
+            .toMatch(/^[0-3][1-9]\/[0-1][1-9]\/20[2-9][0-9], [0-2][0-9]:[0-5][0-9]:[0-5][0-9]$/);
```

Comment:
```suggestion
            .toMatch(/^[0-3][0-9]\/[0-1][0-9]\/20[2-9][0-9], [0-2][0-9]:[0-5][0-9]:[0-5][0-9]$/);
```

otherwise 30-10-2022 won't match. Date regex is hard (impossible even)

Reply:
oops!